includes/html/api_functions.inc.php

```diff
@@ -989,6 +989,15 @@ function list_alerts(\Illuminate\Http\Request $request)
 
     $order = 'timestamp desc';
 
+    
+    $alert_rule = $_GET['alert_rule'];
```

Comment:
please change it to `$alert_rule= $request->get('alert_rule');`

Reply:
done